### DISCUSSION TITLE
Did head.jsx will be deprecated later?

### DISCUSSION THREAD

**Discussion by TotomInc** [vercel/next.js#46403] [Help] [answered]
With the recent changes to how head is handled using the `Metadata` object in 13.2, I wanted to know if the previous `head.jsx/tsx` will still be supported.

I don't want to use the `Metadata` object, all I want to do is just inserting HTML code without having to go through 30 different object properties.

**Comment by icyJoseph** [comment, marked as answer]
JSX is just objects anyway... but I get your sentiment, it is a long list of possibilities: https://github.com/vercel/next.js/blob/5792533781e6d6fdf10d5b400d8e97d25bd5edc3/packages/next/src/lib/metadata/types/metadata-interface.ts

though they are very well documented, but to answer your question specifically:

> Note: Built-in support for SEO through metadata is available on next@canary. This replaces the previous head.js implementation. The [head.js special file](https://beta.nextjs.org/docs/api-reference/file-conventions/head) will be deprecated in 13.2 and **_removed_** in a future version. [Learn more](https://beta.nextjs.org/docs/guides/seo).

![Screenshot 2023-02-27 at 09 28 53](https://user-images.githubusercontent.com/21013447/221512368-72ff2cfe-c89d-4e6e-bb56-4b8b6caa07ec.png)
